test/unit/multiple-versions.js

```diff
@@ -0,0 +1,33 @@
+import helperElement from '../lib/element';
+import helperFixture from '../lib/fixture';
+import helperReady from '../lib/ready';
+
+import skateMaster from '../../src/index';
+// FIXME Babel seems to wreck this one - skate is exposed, not skate014
+import {skate as skate014} from '../skate/0.14.3.js'; // eslint-disable-line no-unused-vars
```

Comment:
You'd expect the exported `skate` symbol to be imported into the local `skate014` symbol, but somehow Babel messes that up and populates a local `skate` symbol. This can't be redefined, so I need to write it into an `oldSkate` symbol. We can fix this with a newer Babel version later, maybe.